internal/controllers/users.go

```diff
@@ -156,12 +156,17 @@ func (s Server) AddUsages(ctx echo.Context) error {
 	if resourceName == "" {
 		return ctx.JSON(http.StatusBadRequest, model.ErrorResponse("Invalid Resource Name", http.StatusBadRequest))
 	}
+	usageValueString := ctx.Param("usage_value")
+	if usageValueString == "" {
+		return ctx.JSON(http.StatusBadRequest, model.ErrorResponse("Invalid Usage Value", http.StatusBadRequest))
+	}
+	UsageValueFloat := ParseFloat(usageValueString)
 	var res = Result{}
 	err := s.GORMDB.Table("user_plans").Select("user_plans.id, users.user_name,plan_quota_defaults.resource_type_id ").Joins("JOIN users on user_plans.user_id=users.id").Joins("JOIN plan_quota_defaults on plan_quota_defaults.plan_id=user_plans.plan_id").Joins("JOIN resource_types on resource_types.id=plan_quota_defaults.resource_type_id").Where("users.user_name=? and resource_types.name=?", username, resourceName).Scan(&res).Error
```

Comment:
I'd really like to have long lines like this split up into multiple lines in the source. Horizontal scrolling is one of my pet peeves.